### DISCUSSION TITLE
Upgrade to latest version?

### DISCUSSION THREAD

**Discussion by pebblecoder5** [Freika/dawarich#362] [General]
For newbies maybe a section in the https://dawarich.app/docs/intro on how to upgrade to the latest version? :D

**Comment by Freika** [comment]
Added it there :)

**Comment by pebblecoder5** [comment]
![image](https://github.com/user-attachments/assets/236e84d6-6e3a-4e87-8da9-c1ddd849b133)
?

**Reply by Freika** [reply]
Hmm it wasn't deployed for some reason, now it is: https://dawarich.app/docs/intro#update-your-dawarich-instance

**Reply by pebblecoder5** [reply]
Maybe its just my version but I have to do:
![image](https://github.com/user-attachments/assets/0e9581ae-6eb9-4d4b-9e8e-477d3ebfe683)

**Reply by Freika** [reply]
Note, `docker-compose` has been deprecated in favor of `docker compose` command (without the dash). Otherwise, it seems you're good and your dawarich instance should have been successfully updated.

Don't forget to pay attention to release notes, as sometimes there are breaking changes.
